### ISSUE TITLE
User-tweaked designs on Prusa's website

### ISSUE THREAD

**Issue by morrowsend** [Charlotte-MEDI/Faceshields#28]
Has anyone tried these?
[This design](https://www.prusaprinters.org/prints/26474-prusa-rc2-no-elastic-required-secure-with-rope-web) uses either string or webbing (think tie-down strap material conference bag handles) instead of elastic.  He mentions an elastic shortage (never heard of such a thing). 

There's also[ this slightly modified version of the masks ](https://www.prusaprinters.org/prints/26507-prusa-protective-shield-remix-for-vase-mode-printi)that uses "Vase Mode" and uses about 1/4th the plastic. Should be faster to print.

You can see all the remixes by scrolling down [this page](https://www.prusaprinters.org/prints/25857-protective-face-shield-rc1/remixes).

Thoughts?

**Comment by JeremyProffitt** [comment]
I've started printing some of those.  The vase mode one scares me for strength, but am trying it.

**Comment by JeremyProffitt** [comment]
I have samples of these printing now

https://www.prusaprinters.org/prints/26077-prusa-medical-face-shield-lite
https://www.prusaprinters.org/prints/26474-prusa-rc2-no-elastic-required-secure-with-rope-web
https://www.prusaprinters.org/prints/26507-prusa-protective-shield-remix-for-vase-mode-printi

**Comment by danjstone** [comment]
Thanks Jeremy! Looking forward to hearing if we should move over to any of these

**Comment by tjfagan** [comment]
Does this change the transparent peice?

**Comment by JeremyProffitt** [comment]
No, however, if a volunteer's printer isn't properly calibrated, they could be printing the piece just enough off the part doesn't fit, it's an issue we've seen.
